packages/designto-flutter/painting/painting-box-decoration.ts

```diff
@@ -33,8 +36,25 @@ function fromBackground(b: Background): BoxDecoration {
   }
 }
 
-function fromGradient(): BoxDecoration {
-  throw "not ready.";
+function fromGradient(g: Gradient): BoxDecoration {
+  switch (g._type) {
+    case GradientType.LINEAR: {
+      return new BoxDecoration({
+        gradient: linearGradient(g),
+      });
+    }
+    case GradientType.RADIAL: {
+      return new BoxDecoration({
+        gradient: radialGradient(g),
+      });
+    }
+    default: {
```

Comment:
+ add case
```
case undefined: {
   return;
}
```

(Just a suggestion)

Reply:
I think there could be a contingency. Added with comments.